cpp/dolfinx/refinement/utils.cpp

```diff
@@ -217,7 +216,7 @@ refinement::create_new_vertices(
   std::map<std::int32_t, std::int64_t> local_edge_to_new_vertex;
   for (int local_i = 0; local_i < edge_index_map->size_local(); ++local_i)
   {
-    if (marked_edges[local_i] == true)
+    if (marked_edges[local_i])
```

Comment:
I'm not sure about this given that `marked_edges` is not `bool`.

Reply:
We use this logic everywhere else in the code. See for instance
https://github.com/FEniCS/dolfinx/blob/a91c7fbb4e61f99b7aa20b519021e38d051a9aab/cpp/dolfinx/refinement/plaza.cpp#L72-L83